### PR TITLE
Feature/pokemon info

### PR DESCRIPTION
**Pull request by Thee-Dust**
## Description
Please include a summary of the change and which issue is fixed. Please also include relevant screenshots of your code being deployed. 
- add basic styling to pokemon info and cards. adds a favoriting button

Fixes # (issue)
- #

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring

## Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/components/PokemonCard/PokemonCard.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import caughtBall from  '../../Assets/caughtBall.png'
 const PokemonCard = ({id, img, name}) => {
   return (
     <Link to={`/${id}`} className="pokemon-card">
-      <img ></img>
+      <img src={uncaughtBall} alt="Pokeball"></img>
```
Comment (rhen92) [inline src/components/PokemonCard/PokemonCard.js]: I love the pokeball image you found so we can favorite it that way!

**Comment by rhen92** [comment]
Good job on the styling!

File: src/components/PokemonCard/PokemonCard.css [hunk 2]
```diff
@@ -1,10 +1,24 @@
 .pokemon-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
   width: 100%;
   border-radius: 20px;
   background: linear-gradient(145deg, #adadad, #cdcdcd);
+  text-decoration: none;
+  color: black;
```
Comment (asiisii) [inline src/components/PokemonCard/PokemonCard.css:10]: thanks for removing the decoration for the pokemon names

File: src/components/PokemonCard/PokemonCard.js [hunk 3]
```diff
@@ -1,17 +1,20 @@
 import React from 'react';
 import './PokemonCard.css'
 import { Link } from 'react-router-dom';
+import uncaughtBall from '../../Assets/uncaughtBall.png';
+import caughtBall from  '../../Assets/caughtBall.png'
 
 const PokemonCard = ({id, img, name}) => {
   return (
     <Link to={`/${id}`} className="pokemon-card">
+      <img className="pokeball" src={uncaughtBall} alt="Pokeball" />
```
Comment (asiisii) [inline src/components/PokemonCard/PokemonCard.js:10]: loved our catch ball

**Review by asiisii — APPROVED** [review]

**Pull request merged**